tests/modules/nf-core/somalier/ancestry/main.nf

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { SOMALIER_EXTRACT  } from '../../../../../modules/nf-core/somalier/extract/main.nf'
+include { SOMALIER_ANCESTRY } from '../../../../../modules/nf-core/somalier/ancestry/main.nf'
+include { UNTAR             } from '../../../../../modules/nf-core/untar/main.nf'
+
+workflow test_somalier_ancestry {
+
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file("https://storage.googleapis.com/genomics-public-data/simons-genome-diversity-project/vcf/LP6005441-DNA_A01.annotated.nh2.variants.vcf.gz", checkIfExists: true),
```

Comment:
Yes especially because this data doesn't seem to be small. The tests are meant to run a quick check if the module actually works in all possible ways

Reply:
We tried using data from the repo, but we found the tests only works on more fleshed out datasets 😞. Since the URL's used are also stable, I don't think it's much of an issue?

Reply:
Okay, fine by me then :), The main problem I think is that these files are probably big?

Reply:
updated with links to files in nf-core tests repository